client/components/.npmignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I believe this is redundant here: https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package